nss-day-cohort-47/tabloidfullstack-fiddlesticks
# Bk delete tags

# Description
added authorization to create tag

## Type of change
Please delete options that are not relevant.
- [x] Delete button is inactive

# Testing Instructions
Navigate to tag management and create a tag, should show on list and be added to database

# Checklist:
